.github/workflows/shared_deploy_cloudhub_1_0.yml

```diff
@@ -37,6 +37,8 @@ on:
         required: false
       NEXUS_PASSWORD:
         required: false
+      NEW_RELIC_API_KEY:
+        required: false
```

Comment:
I'm planning to leave it as `required: false` until we finish setting up `NewRelic` for all apps. Then we can change it into `true` 🤔 

But maybe we can just leave it as `required: false` anyway, since this only affect the newest version, which is `shared_deploy_cloudhub_1_0.yml@v1.7`? 💭

Reply:
It's ok to leave this as required false @tednguyendev as this is the shared action, other team/company might not use NEW_RELIC